src/plugins/scriptedwizard/resources/wxwidgets/wizard.script

```diff
@@ -1038,41 +1034,39 @@ function SetupTarget(target, is_debug)
         if (IsMonolithic)
         {
             target.AddLinkLib(LibPrefix + _T("wxmsw") + LibWxVer + LibUnicSuffix + LibDebugSuffix + LibSuffix);
-            if (!IsDLL)
-            {
-                target.AddLinkLib(LibPrefix + _T("wxpng") + LibDebugSuffix + LibSuffix);
-                target.AddLinkLib(LibPrefix + _T("wxjpeg") + LibDebugSuffix + LibSuffix);
-                target.AddLinkLib(LibPrefix + _T("wxtiff") + LibDebugSuffix + LibSuffix);
-                target.AddLinkLib(LibPrefix + _T("wxzlib") + LibDebugSuffix + LibSuffix);
-            }
+            if (LibWxGL) target.AddLinkLib(LibPrefix + _T("wxmsw") + LibWxVer + LibUnicSuffix + LibDebugSuffix + _T("_gl") + LibSuffix);
```

Comment:
Is it possible to add a function which generates the name and it is more readable?
Something like `expandLibName(name)`

Reply:
I suppose it is, I just wanted to make minimal changes to the wizard. I am not really familiar with Squirrel  nor the wizard API, so refraining from any unneeded changes seemed like a good idea.

TBH, both XRC and script (I did not even look at other files) seem a bit messy, e.g., see several different naming patterns for variables used there.

Reply:
I'd prefer if you add a function... It should be as simple as
```
function name(param) { return globals + param; }
```
and then just call it.
Generally the mantra/goal should be leave the code in better shape then before your arrival.
I can do it before pushing, but I'm mostly running linux, so this would be untested and then you'll have to provide patches to fix it after me.

Reply:
Done in commit "Use helper function to generate names for wxWidgets libraries"